src/ort_genai_c.cpp

```diff
@@ -888,4 +888,12 @@ void OGA_API_CALL OgaDestroyNamedTensors(OgaNamedTensors* p) { delete p; }
 void OGA_API_CALL OgaDestroyAdapters(OgaAdapters* p) { p->ExternalRelease(); }
 void OGA_API_CALL OgaDestroyRuntimeSettings(OgaRuntimeSettings* p) { delete p; }
 
+void OGA_API_CALL OgaRegisterExecutionProviderLibrary(const char* registration_name, const wchar_t* path) {
+  Ort::api->RegisterExecutionProviderLibrary(&(Generators::GetOrtEnv()), registration_name, path);
+}
```

Comment:
Add the new function to [this file](https://github.com/microsoft/onnxruntime-genai/blob/main/src/models/onnxruntime_api.h).

```suggestion
void OGA_API_CALL OgaRegisterExecutionProviderLibrary(const char* registration_name, const char* library_path) {
  Ort::RegisterExecutionProviderLibrary(&(Generators::GetOrtEnv()), registration_name, fs::path(library_path));
}
```